pom.xml

```diff
@@ -65,8 +65,8 @@
   </scm>
 
   <properties>
-    <jenkins.version>1.609.3</jenkins.version>
-    <java.level>6</java.level>
```

Comment:
Lol.

Reply:
Yeaaaaaah. I just bumped it to match `workflow-multibranch`, though it obviously coulda gone higher.

Reply:
We should be able to revert this now. I want to bump, but it will be preceded with an announcement and minor version bump.